### PR TITLE
Msalem/fix scheduler button

### PR DESCRIPTION
**Pull request by msalemcode**
Enforce the scheduler code to check if the feature is enabled.

Test the PR:
1- Enable the metered support flag. Make sure you can access all pages for scheduler including the link from Subscription Managed Usage

2- Disable the metered support flag. Make sure you can not see or access the any scheduler page 

3- try hardcoded url <>/scheduler <>/scheduler/NewScheduler ..etc

4- Enable again and see if you can access the pages

Any other use case please let me know.

### CODE REVIEW

**Review by santhoshbomma9 — COMMENTED** [review]
@msalemcode I turned on the metered scheduler, created an item and turned off. But looks like my meters are still being reported. I think this might a major functionality gap. hence assigning the PR back to you. Please let me know if you want to discuss further.

File: src/AdminSite/Controllers/SchedulerController.cs [hunk 1]
```diff
@@ -81,21 +89,36 @@ public class SchedulerController : BaseController
...
+                List<SchedulerManagerViewModel> getAllSchedulerManagerViewData = new List<SchedulerManagerViewModel>();
+                this.TempData["ShowWelcomeScreen"] = "True";
+                var currentUserDetail = this.usersRepository.GetPartnerDetailFromEmail(this.CurrentUserEmailAddress);
+
+                getAllSchedulerManagerViewData = this.schedulerService.GetAllSchedulerManagerList();
+
+                if (this.saaSApiClientConfiguration.SupportMeteredBilling)
+                {
+                    this.TempData.Add("SupportMeteredBilling", "1");
+                    this.HttpContext.Session.SetString("SupportMeteredBilling", "1");
```
[the remark this answers is not in the document]
Comment (msalemcode) [reply src/AdminSite/Controllers/SchedulerController.cs]: removed

File: src/DataAccess/Migrations/Custom/BaselineV7_Seed.cs
Comment (santhoshb-msft) [inline src/DataAccess/Migrations/Custom/BaselineV7_Seed.cs:1]: @msalem can you revert this if possible please?

**Review by santhoshb-msft — COMMENTED** [review]
Changes observed:
1. doc updates with new way of enabling meter scheduler using db setting
2. added appconfigservice into the base controller and set the logic to enable/disable meter scheduler
3. passed the appconfig service from all controllers 
4. adjusted to show/hide scheduler controller actions based on 2 and other views
5. baseline 741 seed/dessed added for new db setting
6. meter scheduler logic updated based on the dbsetting
7. Tested directed deploy and upgrade scenario,

@msalemcode all looks good except for one step. Lets say 7.4.0 is in use with active scheduler items and someone upgraded to this release and the db setting is by default disabled, which means all the existing scheduler items wont run right. We should go in manually and update the db setting. How should we handle this?
8. 
9. 
10.

**Review by santhoshb-msft — COMMENTED** [review]
@msalemcode  could you validate the below scenario please?

1. deploy 7.4.0
2. create hourly schedule
3. let it run once
4. upgrade to your new PR code
5. Expected, hourly job should run by default
6. Actual: Item didnt get picked up

**Pull request closed without merging**